templates/includes/footer.html

```diff
@@ -5,16 +5,17 @@
         <h3>Contact Info</h3>
         <div class="content">
           <p>
-            OSU Open Source Lab<br />
-            Kerr Admin B211<br />
-            Corvallis, OR 97331<br />
-            General Inquiries:<br /><span style="color: #C34500;"><a href="mailto:info@osuosl.org">info@osuosl.org</a></span><br />
-            Support for Project Infrastructure<br /><span style="color: #C34500;"><a href="mailto:support@osuosl.org">support@osuosl.org</a></span><br />
-            Questions about Donations:<br /><span style="color: #C34500;"><a href="mailto:donations@osuosl.org">donations@osuosl.org</a></span>
+            {{ SITENAME }}-- {{ NAME_ABBREV }}<br />
+            {{ ROOM }} {{ BUILDING }}<br />
+            {{ STREET_ADDRESS }}<br />
+            {% for e in EMAIL %}
```

Comment:
I would hold off on this until Annie merges her PR. The email loop may have to change - since the emails will be contained in an array of dictionaries w/ descriptors.